lib/pod.php

```diff
@@ -143,8 +143,13 @@ function is_main_root () {
 
     function get_subsite_urls () {
       $retvals = array();
+      $menu_api_host = "menu-api";
```

Comment:
Please DRY on this computation. Please introduce a `make_menu_api_url($url_suffix)` function and use it throughout.